pkg/util/cluster/cluster.go

```diff
@@ -399,8 +399,9 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 	oc := api.OpenShiftCluster{
 		Properties: api.OpenShiftClusterProperties{
 			ClusterProfile: api.ClusterProfile{
-				Domain:          strings.ToLower(clusterName),
-				ResourceGroupID: fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", c.env.SubscriptionID(), "aro-"+clusterName),
+				Domain:               strings.ToLower(clusterName),
+				ResourceGroupID:      fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", c.env.SubscriptionID(), "aro-"+clusterName),
+				FipsValidatedModules: "Enabled",
```

Comment:
nit  for consistency.  Should be the last one.  
```suggestion
				FipsValidatedModules: api.FipsValidatedModulesEnabled,
```